benchmark/opperf/rules/default_params.py

```diff
@@ -31,6 +31,7 @@
 
 # For Unary operators like abs, arccos, arcsin etc..
 DEFAULT_DATA = [(1024, 1024), (10000, 1), (10000, 100)]
+DEFAULT_DTYPE = ['int32'] # required parameter for amp_cast, cast
```

Comment:
Can we also add `float32` here?

Reply:
We can't because cast needs it to be int compulsorily.

Reply:
I ran both ops (`cast` and `amp_cast`) with `float32` in the shell and they worked properly.

Reply:
I stand corrected. Fixed it now.